modules/storage-configuration/src/main/java/org/apache/ignite/internal/storage/configurations/StorageEngineConfigurationSchema.java

```diff
@@ -17,16 +17,11 @@
 
 package org.apache.ignite.internal.storage.configurations;
 
-import org.apache.ignite.configuration.annotation.PolymorphicConfig;
-import org.apache.ignite.configuration.annotation.PolymorphicId;
+import org.apache.ignite.configuration.annotation.Config;
 
 /**
  * Storage engine configuration schema.
  */
-@PolymorphicConfig
+@Config
```

Comment:
Why is this change needed?

Reply:
Because it was replaced by extensions approach, please look at https://github.com/gridgain/apache-ignite-3/pull/92#discussion_r1477878029